omnibus_build.sh

```diff
@@ -35,6 +37,12 @@ if [ -n "$RPM_SIGNING_PASSPHRASE" ]; then
   gpg --import /keys/RPM-SIGNING-KEY.private
 fi
 
+git --git-dir=/integrations-core fetch --all || true
+git --git-dir=/integrations-core checkout $INTEGRATION_CORE_BRANCH || true
+
+echo "integrations"
+ls /integrations-core
```

Comment:
let's remove this from here, if we want to log something about the integrations we're building I think we should do it from the `build` step of `datadog-agent-integrations` with `Omnibus.logger.debug("")` for instance